modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java

```diff
@@ -684,7 +684,7 @@ public JSONObject getFormLocalizedDescriptionJSONObject()
 		DDMFormInstance formInstance = getDDMFormInstance();
 
 		if (formInstance == null) {
-			jsonObject.put(getDefaultLanguageId(), "");
+			jsonObject.put(getDefaultLanguageId(), getFormDescription());
```

Comment:
This fix seems to be done by surgeons :man_health_worker:, boys! Good job! :wink: